funders/nsf-grants/geo-ose.md

```diff
@@ -5,21 +5,19 @@
 
 GEO OSE seeks to build upon  the POSE solicitation by targeting community building programs specific to the geospace sciences, ocean sciences, Earth sciences, and polar sciences. As per the solicitation:
 
-
```

Comment:
These edits below were due to the `npm run lint` command failing as well. I don't see any specific github action that runs this command though. @RichardLitt do you know if we use this anymore? I may just be missing where it is run.

Reply:
Take a look at package.json, and build.sh. If there is something missing, we should open a new issue and document this in the README.